assets/js/item_details.js

```diff
@@ -53,6 +59,29 @@ function addQuantity(value, caller) {
     );
 }
 
+$(".item_quantity").focusout(e => {
+    toggleDeleteButton($(e.target))
+})
+
+
+function toggleDeleteButton(target) {
+    console.log(target)
+    let targetVal = $(target).siblings(".item_quantity").addBack().val()
+    console.log(targetVal)
+    if (targetVal > 0) {
+        console.log($(target).siblings(".deleteRow"))
+        $(target).siblings(".deleteRow").hide()
```

Comment:
![image](https://user-images.githubusercontent.com/19233251/80316994-18117a00-87cf-11ea-87ae-32c676a9c587.png)
I lost my original comment for some reason but have a picture of it

Reply:
> 1. the button isn't a row it it just declared as , the "." prefix references classes in JQuery selectors.
> 
> 1. the elements seem to jump around anyway as a result of other rows being added. I don't know if it's worth doing this, since I just implemented it in this way already.

copy and pasted the reply from the original comment